pkg/apis/flyteworkflow/v1alpha1/node_status.go

```diff
@@ -128,7 +128,9 @@ func (in *DynamicNodeStatus) SetDynamicNodePhase(phase DynamicNodePhase) {
 }
 
 func (in *DynamicNodeStatus) SetExecutionError(err *core.ExecutionError) {
-	in.Error = &ExecutionError{ExecutionError: err}
+	if err != nil {
```

Comment:
If it's nil, should we set `in.Error = nil` ?

Reply:
hmm interesting point. Like a reset?